sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx

```diff
@@ -62,6 +84,7 @@ const Repay: React.FC<RepayProps> = ({ loan, loanId, loanTypeIsETH, loanContract
 						onTxConfirmed: () => {},
 					}),
 			});
+			router.push('/loans/list');
 		} catch {
```

Comment:
guess you can remove the catch?

Reply:
I take a look and try to handle errors properly